core/client/fs/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -163,8 +163,7 @@ public static void persistAndWait(final FileSystem fs, final AlluxioURI uri,
         LOG.debug("Polling file system master for the status of {}", uri);
         return fs.getStatus(uri);
       } catch (Exception e) {
-        LOG.debug("Exception when attempting to getStatus while waiting for persistence {}",
-            e.toString());
+        LOG.debug("Exception when attempting to getStatus while waiting for persistence", e);
```

Comment:
For a throwable object, it cannot instead of `toString`.

![image](https://user-images.githubusercontent.com/17329931/150792101-9b121b82-6160-4b5f-9046-9a4ced952af1.png)

But for this case, i think it would be better to print the exception call stack, so that it can be easy to find the root cause of that exception